test/cypress/e2e/tasks-page.cy.js

```diff
@@ -1,4 +1,35 @@
 describe('Tasks page', () => {
+  beforeEach(() => {
```

Comment:
Is this executed for every new `it` block? If so, seems gratuitous

Reply:
State resets between tests, so it's mandatory. That said, best practice according to cypress is to directly inject the login values to local storage to make it faster, so we could update the login command to use that method and make it faster.

Reply:
Only affects speed of test suite, nonblocking